public/data/tanishagupta1.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Tanisha Gupta",
+  "bio": "Software Developer| Tech-Writer | Open Source Enthusiast",
```

Comment:
```suggestion
  "bio": "Software Developer | Tech-Writer | Open Source Enthusiast",
```

Reply:
Yes for Sure, Sorry for the inconvenience.